internal/server/app_vars.go

```diff
@@ -10,6 +11,7 @@ type AppVars struct {
 	XSRFToken       string
 	Tabs            []Tab
 	EnvironmentVars EnvironmentVars
+	Errors          model.ValidationErrors
```

Comment:
kind of weird having both `Error` and `Errors` fields, would be nice if they could be combined somehow, maybe? Or rename this to `ValidationErrors` to make it clear it is a different type of thing